src/components/Todo/TodoItem.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[];
+  modalInfo: Todo | null;
+  setModalInfo: (todo: Todo) => void;
+};
+
+export const TodoItem: React.FC<Props> = ({
```

Comment:
`TodoItem` is a single means that it is single todo, but you render a list here 🤷